src/components/cafes/[id]/BookMark/index.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import Bookmark from '@/assets/Icon/Bookmark.svg';
+import { bookMark } from './BookMark.css';
+
+interface BookMarkProps {
+  cafeIdForBookMark: string;
+}
+
+export default function BookMark({ cafeIdForBookMark }: BookMarkProps) {
+  const [bookMarks, setBookMarks] = useState<string[]>([]);
```

Comment:
아래처럼 표현하면 `string`보다 명확히 표현될 것 같아용
```suggestion
type CafeId = string

export default function BookMark({ cafeIdForBookMark }: BookMarkProps) {
  const [bookMarks, setBookMarks] = useState<CafeId[]>([]);
```